scrapy/czlscrape/utils.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+
+def guess_publication_type(text, rules):
+    text = text.lower()
+    text = re.sub(r'[șş]', 's', text)
+    text = re.sub(r'[țţ]', 't', text)
+    text = re.sub(r'[ăâ]', 'a', text)
+    text = re.sub(r'[î]', 'i', text)
+
+    for substr, publication_type in rules:
```

Comment:
Parțial off-topic și nu prea: spiderele nu ar trebui să se ocupe de logica asta. Guessing-ul să se facă la receiver, unificat.
Spiderele pot în schimb să trimită `type` atunci când îl știu cu siguranță din ce-au scrapat.

[romgleză cu diacritice. for the win.]